pages/access/service/[serviceId].js

```diff
@@ -510,7 +469,7 @@ function MarketplaceService() {
     }
 
     return columnDefinition;
-  }, [serviceId, selectedResource, resourceInstanceList]);
+  }, [serviceId, selectedResource, filteredInstances]);
```

Comment:
Filtered instances has been added here, but doesn't look like we are using this variable